docs/core/docker/build-container.md

```diff
@@ -22,7 +22,7 @@ You'll learn to:
 
 You'll understand the Docker container build and deploy tasks for a .NET Core application. The *Docker platform* uses the *Docker engine* to quickly build and package apps as *Docker images*. These images are written in the *Dockerfile* format to be deployed and run in a layered container.
 
-> [!TIP]
+> [!WARNING]
 > If you're working with an existing ASP.NET Core application, see the [Learn how to containerize an ASP.NET Core application](/aspnet/core/host-and-deploy/docker/building-net-docker-images) tutorial.
```

Comment:
Since it's a warning, I think it would be helpful to add another sentence as to why the user should take a look at that tutorial?

Reply:
Maybe "instead of this one" at the end?